oauth2client/contrib/django_util/__init__.py

```diff
@@ -299,7 +403,11 @@ def has_credentials(self):
     @property
     def credentials(self):
         """Gets the authorized credentials for this flow, if they exist"""
-        return storage.get_storage(self.request).get()
+        if oauth2_settings.storage_model:
```

Comment:
Explicitly compare to `None`, e.g, `if oauth2_settings.storage_model is not None:`